### DISCUSSION TITLE
Not working

### DISCUSSION THREAD

**Discussion by dbcoyer** [vslinko/obsidian-outliner#87] [Archive]
I think I have followed the instructions exactly. I have even uninstalled and reinstalled Obsidian and started a new vault with only this plugin installed.
Yet, I cannot see that it is doing anything when I try to create an outline. I use tab to indent an item but there is no change from the base Obsidian implementation. 
Any help would be greatly appreciated. Thank you.
EDIT: using obsidian 0.11.13 and version 1.1.4 of the add in.

**Comment by dbcoyer** [comment]
Never mind. I figured out my dumb mistake. Too embarrassing to even reveal here. Pretend like you never saw this. All is well.

**Reply by vslinko** [reply]
@dbcoyer no problem, it’s okay to make mistakes :) have a good day!

**Reply by vincentteoh** [reply]
hi, I have the same problem and can't figure it out.

**Reply by dbcoyer** [reply]
Now you're forcing me to admit my dumb mistake. ;)

@vincentteoh I had forgotten to use markdown to start the outline. I was creating tabbed indents but without markdown list elements. If you are doing the same, try this:
- TLI
<tab> - Child 1
<tab> - Child 2

IOW, use a dash (-) or an asterisk (*) followed by a space and then the text. Then, for children, tab (or use 4 spaces) and the same dash or asterisk. 

Hope that helps.

**Reply by vincentteoh** [reply]
Thanks and it works now! At least you found out the solution!
